extensions/blocks/podcast-player/index.js

```diff
@@ -24,6 +24,12 @@ export const settings = {
 	icon: queueMusic,
 	category: 'jetpack',
 	keywords: [],
+
+	styles: [
+		{ name: 'small', label: __( 'Small' ), isDefault: true },
+		{ name: 'large', label: __( 'Large' ) },
+	],
```

Comment:
Are these functional?

Reply:
Yes, but the styles aren't implemented yet. You can select the styles from the sidebar as well.

<img src="https://user-images.githubusercontent.com/77539/76799338-10ae8800-67b0-11ea-83a9-274c9546478d.png" width="500px" />

Reply:
Could we either hold off on that until we have styles or add styles to make it functional?

Reply:
...and it the styles add a load of extra work not directly related to "settings" then we could consider doing this in a follow up PR.

Reply:
> Could we either hold off on that until we have styles or add styles to make it functional?

I think it's part of the development process. This is not the only change that will keep in a middle state among all PRs/patches that we are doing day by day. The scope of this PR is adding the block settings in the sidebar.

> ...and it the styles add a load of extra work not directly related to "settings" then we could consider doing this in a follow up PR.

Why do we need to discard something that we are going to use in the near future? I prefer to discard it in the future (if something unexpected happens) than now.
I've based the progress of this PR according to the [given design](https://github.com/Automattic/Dotcom-roadmap/issues/100). The lack of designs is something that we've lamented in previous iterations.

I prefer to keep these changes here and keep iterating in other patches.